### ISSUE TITLE
Format code with 4 spaces

### ISSUE THREAD

**Issue by nmtitov** [dart-lang/dart_style#1425]
Hi guys,
I'd like to ask how to configure Dart code formatter to put 4 spaces instead of 2. 
I'm working in Android Studio from Jetbrains.
Any advice?

**Comment by munificent** [comment]
The formatter doesn't support configuring indentation. See [here](https://github.com/dart-lang/dart_style/wiki/FAQ#why-cant-i-configure-it) for more.

**Comment by munificent** [comment]
It's a very deliberate design choice and we have no plans to add any significant level of configurability to the formatter.

**Comment by nmtitov** [comment]
@munificent after a quick googling I found the "north korean dictator" discussion so I guess this won't be changed indeed :)

**Comment by munificent** [comment]
Yeah. I understand this is frustrating to some users, but a fine-grained personally configurable formatter is just fundamentally not the charter for the tool. Its goals are ecosystem-wide consistency and getting users to *not* think about formatting. Lots of configurability are in opposition to both of those goals.

**Comment by nmtitov** [comment]
Yes, I understand the intention to manage huge codebase. I will try to fork the project or find another way around. Thank you.
